package.json

```diff
@@ -19,50 +18,43 @@
       "types": "./dist/types/index.d.ts",
       "node": "./dist/cjs/index.js",
       "require": "./dist/cjs/index.js",
-      "es2015": "./dist/esm/index.js",
-      "default": "./dist/esm5/index.js"
+      "default": "./dist/esm/index.js"
```

Comment:
There is no `"type": "module"` in `package.json`, so the file specified in `"default"` will be parsed as CommonJS, right? But there is a file with ES modules.

```json
".": {
  "types": "./dist/types/index.d.ts",
  "node": "./dist/cjs/index.js",
  "import": "./dist/esm/index.js",
  "default": "./dist/cjs/index.js"
},
```

Reply:
Also, maybe you should move [`"node"`](https://nodejs.org/api/packages.html#conditional-exports) down or remove it altogether?

> `"node"` - matches for any Node.js environment. Can be a CommonJS or ES module file. _In most cases explicitly calling out the Node.js platform is not necessary._

```js
import {timer} from 'rxjs'; // version from current PR

timer(2000).subscribe(() => {
  console.trace('2000');
});
```

```
% node main.js

Trace: 2000
    at Object.next (file:///example/main.js:4:11)
    at ConsumerObserver.next (example/node_modules/rxjs/dist/cjs/internal/Subscriber.js:99:33)
    at Subscriber._next (example/node_modules/rxjs/dist/cjs/internal/Subscriber.js:70:26)
    at Subscriber.next (example/node_modules/rxjs/dist/cjs/internal/Subscriber.js:41:18)
    at AsyncAction.work (example/node_modules/rxjs/dist/cjs/internal/observable/timer.js:28:28)
    at AsyncAction._execute (example/node_modules/rxjs/dist/cjs/internal/scheduler/AsyncAction.js:79:18)
    at AsyncAction.execute (example/node_modules/rxjs/dist/cjs/internal/scheduler/AsyncAction.js:67:26)
    at AsyncScheduler.flush (example/node_modules/rxjs/dist/cjs/internal/scheduler/AsyncScheduler.js:38:33)
    at listOnTimeout (node:internal/timers:573:17)
    at process.processTimers (node:internal/timers:514:7)
```

NodeJS imports the CommonJS (cjs) version instead of the ES version.

Reply:
Honestly, I didn't change anything that was already working. It's still working, this is the minimal change to get the right JS target, I don't want to change it unless it's breaking someone.

Reply:
Agree. I will try to do a thorough research and make a separate issue for this.